src/constants.jl

```diff
@@ -0,0 +1,7 @@
+import Base: @irrational
+import IrrationalConstants: logtwo
+
+@irrational logten 2.302585092994046 log(big(10))
+
+@irrational twoinvsqrtπ 1.1283791670955126 2 / sqrt(big(π))
+@irrational halfsqrtπ   0.886226925452758 sqrt(big(π)) / 2
```

Comment:
I don't _really_ need these. I've made a PR to IrrationalConstants.jl to include `logten`, once that's gone through we can remove this file completely.

Reply:
The PR was merged and released in IrrationalConstants 0.1.1